native/python/pyjp_value.cpp

```diff
@@ -287,6 +286,13 @@ void PyJPValue_assignJavaSlot(JPJavaFrame &frame, PyObject* self, const JPValue&
 	// GCOVR_EXCL_STOP
 
 	JPValue* slot = (JPValue*) (((char*) self) + offset);
+	// GCOVR_EXCL_START
+	// This is an sanity check that should never trigger in normal operations.make
```

Comment:
a sanity check (sorry for being anal)

Reply:
It originally said "an idiot check" check, but my save censor converted it to "an sanity check".  Guess I need to add "an idiot" to the filter first.

I am unfortunately beyond dyslexic, as you may have noticed, resulting in anything that isn't being compiled (and thus checked by a computer) to often result in odd mispellings, weird word choice, and my favorite hilarious word substitutions.  I try to catch them by rereading what I write a day later as usually some distance allows me to interpret the words more clearly, but that still means I miss far more than one would expect from a native speaker.